examples/run-wasm/Cargo.toml

```diff
@@ -10,3 +10,6 @@ repository.workspace = true
 
 [dependencies]
 cargo-run-wasm = "0.3.2"
+
+[package.metadata.release]
+release = false
```

Comment:
These shouldn't be needed as we already have `publish = false`?

Reply:
`cargo release` doesn't respect `publish = false` unfortunately
(which feels like an oversight on their part?)

Reply:
That seems...weird? I could have sworn it respects it when I used it in the past. This seems to imply they are not published? https://github.com/crate-ci/cargo-release/issues/877

Reply:
Yea it won't release (at least I think it won't), but cargo release will still bump the version and such, which is just totally unnecessary for internal crates versioned at "0.0.0".